pyonepassword/pyonepassword.py

```diff
@@ -56,6 +59,44 @@ def __init__(self, op_path=OP_PATH, signin_address=None, email_address=None,
         else:
             self.token = self._do_normal_signin(password)
 
+    @staticmethod
```

Comment:
as with PR #1, for now I'm sticking with my current style of not making methods static.